Source/DafnyLanguageServer/Workspace/DocumentManager.cs

```diff
@@ -75,26 +73,15 @@ public class DocumentManager {
   private const int MaxRememberedChanges = 100;
   private const int MaxRememberedChangedVerifiables = 5;
   public void UpdateDocument(DidChangeTextDocumentParams documentChange) {
-    // According to the LSP specification, document versions should increase monotonically but may be non-consecutive.
-    // See: https://github.com/microsoft/language-server-protocol/blob/gh-pages/_specifications/specification-3-16.md?plain=1#L1195
-    var oldVer = Compilation.TextBuffer.Version;
-    var newVer = documentChange.TextDocument.Version;
-    var documentUri = documentChange.TextDocument.Uri;
-    if (oldVer >= newVer) {
```

Comment:
Why removing this version check? Seemed useful.

Reply:
It's moved to LanguageServerFileSystem